docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.1-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=smallboard_user
+      - POSTGRES_PASSWORD=smallboard_password
+      - POSTGRES_DB=smallboard
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/usr/src/smallboard/
+    ports:
+      - 8000:8000
+    env_file:
+      - ./.env.docker
```

Comment:
Why not provide environment variables via `environment` as for the `db` container above? That way, all of the configuration can stay in this one file.

`.env` seems to be a convenience for local development, but shouldn't we be using env vars for production-like use cases like Docker?

Reply:
Lol that's a much better idea (you can tell this is the first docker config I've written)

Reply:
I couldn't tell! It seems to work great for me (with some minor tweaks)